File: sprint3/By languages/C#/lesson2code4.cs
public static string AsBinaryString(int n)
{
    if (n < 0)
        return "-" + AsBinaryString(-n);
    if (n == 0 || n == 1)
        return n.ToString();
    int lastDigit = n % 2;
    return AsBinaryString(n / 2) + lastDigit.ToString();
}

File: sprint3/By languages/swift/lesson2code4.swift
func asBinaryString(_ n: Int) -> String {
    if n < 0 {
        return "-" + asBinaryString(-n)
    }
    if n == 0 || n == 1 {
        return String(n)
    }
    let lastDigit = n % 2
    return asBinaryString(n / 2) + String(lastDigit)
}
